CompetitionTemplate/src/competition/subsystems/collector/CollectorSubsystem.java

```diff
@@ -19,16 +24,31 @@
     
     @Inject
     public CollectorSubsystem(WPIFactory factory, XPropertyManager propMan){
-        collectorMotorLeft = factory.getSpeedController(0);
-        collectorMotorLeft.setInverted(true);
-        ballExistsSensor = factory.getDigitalInput(1);
-        ballInCollector = propMan.createEphemeralProperty("BallInCollector", false);
+        this.collectorMotorLeft = factory.getSpeedController(0);
+        this.collectorMotorLeft.setInverted(true);
+        this.ballExistsSensor = factory.getDigitalInput(1);
+        this.ballInCollector = propMan.createEphemeralProperty("BallInCollector", false);
+
+        intakePower = propMan.createPersistentProperty("Collector intake power", 1.0);
+        ejectPower = propMan.createPersistentProperty("Collector eject power", -1.0);
     }
     
-    public void setIntakePower(double power) {
+    public void setCollectorPower(double power) {
         this.collectorMotorLeft.set(power);
     }
     
+    public void startEject() {
```

Comment:
remove start from startEject() and startIntake() - just Intake and Eject are fine.